scripts/local/.env

```diff
@@ -0,0 +1,52 @@
+# -----------------------------------------------------------------------------
+# Required Environment Variables for Local setup
+# -----------------------------------------------------------------------------
+#
+# Please locate and paste your string values here to set up 
+# 
+# DO NOT add quotation marks to values
+# -----------------------------------------------------------------------------
+
+# -----------------------------------------------------------------------------
+# IoTHub connection string:
+# {Your IoT Hub} > Shared access policies > Connection string -- primary key
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_ACCESS_CONNSTRING=
+PCS_IOTHUB_CONNSTRING=
+
+# -----------------------------------------------------------------------------
+# IoTHub Info:
+# {Your IoT Hub} > Endpoints > events
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_HUB_NAME=
+PCS_IOTHUBREACT_HUB_ENDPOINT=
+PCS_IOTHUBREACT_HUB_PARTITIONS=
+
+# -----------------------------------------------------------------------------
+# Storage Account information:
+# {Your storage account} > Access keys
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT=
+PCS_IOTHUBREACT_AZUREBLOB_KEY=
+
+# -----------------------------------------------------------------------------
+# Storage Endpoint Suffix: 
+# {Your storage account} > Properties > Primary Blob Service Endpoint > value
+# in the URI after the "blob." (eg. `core.windows.net`)
+# -----------------------------------------------------------------------------
+PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX=
+
+# -----------------------------------------------------------------------------
+# DocumentDb Conn String:
+# {Your Document DB} > Keys > Primary Connection String
+# -----------------------------------------------------------------------------
+PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING=
+PCS_TELEMETRY_DOCUMENTDB_CONNSTRING=
+PCS_TELEMETRYAGENT_DOCUMENTDB_CONNSTRING=
+
+# -----------------------------------------------------------------------------
+# Default setup for local development
+# -----------------------------------------------------------------------------
+PCS_BINGMAP_KEY=static
```

Comment:
should we allow non static maps? and explain how to find the key?

Reply:
directions for map key added.